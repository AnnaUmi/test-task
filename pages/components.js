import React from 'react';

const ComponentsPage = () => <div>component page representation</div>;

export default ComponentsPage;
