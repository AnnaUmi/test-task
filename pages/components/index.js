import React from 'react';

export default () => <div>index components page representation</div>;
