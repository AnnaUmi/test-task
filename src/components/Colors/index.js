/* eslint import/prefer-default-export: off */
export { default as Colors } from './Colors';
