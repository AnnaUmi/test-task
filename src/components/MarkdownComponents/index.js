import Content from './Content.styles';
import Code from './Code';
import InlineCode from './InlineCode';
import Pre from './Pre';
import { TableComp, THElement, THeadElement, TRElement, TDElement } from './Table';

export const MarkdownContent = Content;

export default {
  pre: Pre,
  code: Code,
  inlineCode: InlineCode,
  table: TableComp,
  thead: THeadElement,
  th: THElement,
  td: TDElement,
  tr: TRElement,
};
