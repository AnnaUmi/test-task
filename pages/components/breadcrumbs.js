import React from 'react';
import Document from '../../src/md/breadcrumbs.mdx';
import components from '../../src/components/MarkdownComponents';

export default () => <Document components={components} />;
