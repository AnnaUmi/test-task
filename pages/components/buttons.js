import React from 'react';
import Document from '../../src/md/buttons.mdx';
import components from '../../src/components/MarkdownComponents';

export default () => <Document components={components} />;
