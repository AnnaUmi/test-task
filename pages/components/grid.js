import React from 'react';
import { colors } from '@ticketmaster/prism';
import Document from '../../src/md/grid.mdx';

import components from '../../src/components/MarkdownComponents';

export const contentStyles = {
  backgroundColor: colors.azure.base,
  textAlign: 'center',
};

export default () => <Document components={components} />;
