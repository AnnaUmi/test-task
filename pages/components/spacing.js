import React from 'react';
import Document from '../../src/md/spacing.mdx';

import components from '../../src/components/MarkdownComponents';

export default () => <Document components={components} />;
