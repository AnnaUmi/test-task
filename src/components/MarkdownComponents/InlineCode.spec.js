import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';

import components from './index';

describe('InlineCode', () => {
  it('should match snapshot', () => {
    const tree = renderer.create(<components.inlineCode />);

    expect(tree.toJSON()).toMatchSnapshot();
  });
});
