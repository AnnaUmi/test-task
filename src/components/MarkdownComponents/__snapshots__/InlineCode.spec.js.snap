// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InlineCode should match snapshot 1`] = `
<code
  id="codes"
  style={
    Object {
      "color": "purple",
    }
  }
/>
`;
